dbdev/app.py

```diff
@@ -0,0 +1,74 @@
+from flask import Flask,request,jsonify
+import json
+from flask_restplus import Api,Resource,fields,reqparse
+from models import DbManager
+from flask_jwt_extended import JWTManager,create_access_token
+from werkzeug.security import check_password_hash
+#create an instance of flask
+app=Flask(__name__)
+api=Api(app)
+jwt = JWTManager(app)
+
+"""Add a secret key for the app"""
+app.secret_key = '\xaa\x98\xfb\xf7\xcb\xce\xd3\xdf\x96'
+app.config['JWT_SECRET_KEY'] = '\xe7\x06K\x86>\xe5\x98/\x11\x06\xfbJA-\x86'
+
+class Manage_rides(object):
+    parser = reqparse.RequestParser()
+
+    def __init__(self):
+        pass
+
+    def get_user_login(self):
+
+        self.parser.add_argument('email', required=True,
+                            help="email cannot be blank!")
+        self.parser.add_argument('password', required=True,
+                            help="password cannot be blank!")
+        
+        self.args = self.parser.parse_args()
+
+        return self.args
+
+
+
+R=Manage_rides()
+
+@api.route('/auth/signin')
+class Signin(Resource):
+    def post (self):
```

Comment:
The function name is not descriptive enough. It would be nice to have a single line comment. A comment within a function should at least state the parameters and return value expected.